OpenSim/Simulation/Model/ContactHalfSpace.cpp

```diff
@@ -77,8 +77,14 @@ void ContactHalfSpace::generateDecorations(bool fixed, const ModelDisplayHints&
     const auto& X_BF = getFrame().findTransformInBaseFrame();
     const auto& X_FP = getTransform();
     const auto X_BP = X_BF * X_FP;
-    geometry.push_back(SimTK::DecorativeBrick(Vec3{.005, 0.5, 0.5})
-        .setTransform(X_BP).setScale(1)
+    const double brickHalfThickness = 0.005;
```

Comment:
The issue here is that we are representing a plane with a brick. I think it is misleading to present a plane with a block with thickness. While I agree using the top surface of the brick is better (than the middle) in reality we shouldn't be presenting any thickness. Ideally we could draw planes. For the time being we could use a thickness of 1mm instead of 1cm?

Reply:
I've changed the thickness to 1mm. Here's a comparison:


1cm:

<img width="912" alt="screen shot 2018-07-13 at 11 35 00 am" src="https://user-images.githubusercontent.com/846001/42708025-d17480ce-8690-11e8-9d02-bbab735efc31.png">

1mm:

<img width="912" alt="screen shot 2018-07-13 at 11 34 15 am" src="https://user-images.githubusercontent.com/846001/42708033-da104a24-8690-11e8-84b6-c8d6dc7491bf.png">

Reply:
:+1: Looks more plane like to me.

Reply:
Ah right. Updated.